### ISSUE TITLE
About loss function

### ISSUE THREAD

**Issue by AlexNmSED** [jrzaurin/pytorch-widedeep#72]
Hello everyone.
Can I customize the loss function? Train the model?

good luck

**Comment by francescopisu** [comment]
Hi Alex,
take a look at the documentation of the Trainer class. You see there's a **custom_loss_function** parameter which can be set to a callable of your liking. Additionally, [here](https://github.com/jrzaurin/pytorch-widedeep/blob/master/examples/07_Custom_Components.ipynb) you can take a look at an example where a custom loss function is specified for training.

**Comment by AlexNmSED** [comment]
Thank you very much for your help.

good luck

**Comment by jrzaurin** [comment]
@AlexNmSED can we close this?

**Comment by AlexNmSED** [comment]
Yes，you can.

Thank you
